cob_description/urdf/cob4_arm/cob4_arm.urdf.xacro

```diff
@@ -0,0 +1,342 @@
+      <xacro:default_inertial/>
+  <!--
+      <xacro:cylinder_inertial radius="0.056" length="0.12" mass="2.0">
+        <origin xyz="0 0 ${0.12/2}" rpy="0 0 0" />
+      </xacro:cylinder_inertial>
+  -->
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://cob_description/meshes/cob4_arm/arm_base_link.stl" />
+        </geometry>
+      </visual>
+
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://cob_description/meshes/cob4_arm/arm_base_link.stl" />
+        </geometry>
+      </collision>
+    </link>
+
+    <!-- joint between arm_base_link and arm_1_link -->
+    <joint name="${name}_1_joint" type="revolute">
+      <origin xyz="0 0 0.139" rpy="0 0 0" />
+      <parent link="${name}_base_link"/>
+      <child link="${name}_1_link" />
+      <axis xyz="0 0 1"/>
+      <dynamics damping="0" friction="0"/>
+      <limit effort="81.5" velocity="0.66" lower="${-4.71 + safety_offset}" upper="${4.71 - safety_offset}"/>   <!--Theoretical limit +/-270 , Real Limit +/- 270-->
+      <!--safety_controller k_position="20" k_velocity="50" soft_lower_limit="${-M_PI + 0.01}" soft_upper_limit="${M_PI - 0.01}" /-->
+    </joint>
+
+    <!-- prl100 rotor + shoulder flansh -->
+    <link name="${name}_1_link">
+      <xacro:default_inertial/>
+<!--
+      <inertial>
+        <origin xyz="0.0 0.0662 0.0473" rpy="0 0 0"/>
+        <mass value="0.8"/>
+        <inertia ixx="0.002034" ixy="0.0" ixz="0.0" iyy="0.001437" iyz="0.000784" izz="0.00161" />
+      </inertial>
+-->
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://cob_description/meshes/cob4_arm/arm_1_link.dae" />
+        </geometry>
+      </visual>
+
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://cob_description/meshes/cob4_arm/arm_1_collision.stl" />
+        </geometry>
+      </collision>
+    </link>
+
+    <!-- joint between arm_1_link and arm_2_link -->
+    <joint name="${name}_2_joint" type="revolute">
+      <origin xyz="0 0 0.0915" rpy="${M_PI/2} 0 0"/>
+      <parent link="${name}_1_link"/>
+      <child link="${name}_2_link"/>
+      <axis xyz="0 0 1"/>
+      <dynamics damping="0" friction="0"/>
+      <limit effort="48" velocity="1.57" lower="${-1.91 + safety_offset}" upper="${1.91 - safety_offset}"/>   <!--Theoretical limit +/-170 , Real Limit +/- 110 (self collision)-->
+      <!--safety_controller k_position="20" k_velocity="50" soft_lower_limit="${-2.0943 + 0.01}" soft_upper_limit="${2.0943 - 0.01}" /-->
+    </joint>
+
+    <link name="${name}_2_link">
+      <xacro:default_inertial/>
+<!--
+      <inertial>
+        <origin xyz="0 0.018 -0.018" rpy="0 0 0"/>
+        <mass value="3.9"/>
+        <inertia ixx="0.009780" ixy="0.0" ixz="0.0" iyy="0.013300" iyz="0.0" izz="0.009780" />
+      </inertial>
+-->
```

Comment:
no, those are the real inertia...but gazebo is not able to properly simulate them...but might be able to in the future...